### ISSUE TITLE
Use MixMatch on tabular data.

### ISSUE THREAD

**Issue by Julianbaozi** [google-research/mixmatch#28]
I'm right now trying MixMatch on my tabular data. But I don't know what would be a good augmentation method. Could you please give me some advice? Thank you!

**Comment by david-berthelot** [comment]
You can use it without augmentation. MixUp, which is baked in the method, might be sufficient for your data.

**Comment by Julianbaozi** [comment]
Thank you for your response. That is very helpful.

**Comment by Julianbaozi** [comment]
Should I do cross validation? I have only 360 labeled points. The validation set should only take labeled data and would have around 90 points (4 folds). I'm I correct?

**Comment by david-berthelot** [comment]
You'll need a test set (or take from your labeled examples to make on if you don't have one) to assess the performance of the model. That's always a requirement. Now as to the proportion to keep for training and for testing, I don't know what it should be. I would experiment with various values (say 50/50, 75/25, and so on) and pick the smallest test set that works.

**Comment by ghost** [comment]
Hello, it's not that semi supervised learning like miamache can't be used on unbalanced data sets. I use it on my own data set. The test set is accurate, and the test set rises by 2 epochs and then decreases continuously. However, the loss of training set is declining. What's the matter? Is my dataset too unbalanced

**Comment by Julianbaozi** [comment]
I think your test set might be biased.
Or you can try to fine tune the parameters
